include/GafferImageUI/ImageGadget.h

```diff
@@ -144,11 +144,24 @@ class GAFFERIMAGEUI_API ImageGadget : public GafferUI::Gadget
 			Complete
 		};
 
+		enum class BlendMode
+		{
+			Replace,
+			Over,
+			Under,
+			Difference
+		};
+
+		void setBlendMode( BlendMode blendMode );
+		BlendMode getBlendMode() const;
+
 		State state() const;
 		ImageGadgetSignal &stateChangedSignal();
 
 		Imath::V2f pixelAt( const IECore::LineSegment3f &lineInGadgetSpace ) const;
 
+		void setWipe( bool enabled, const Imath::V2f &position, const Imath::V2f &direction );
```

Comment:
Switched to individual accessors.

Reply:
It'd be very useful if you could link to the commit with the fix, to aid navigation for the reviewer. In this case, navigation is particularly hard because you've rebased and all the commits - including the originals - have moved below the line of the last review. I'd also recommend using `git commit --fixup` so that later squashing becomes trivial via `git rebase -i --autosquash`, and its doubly clear what is new and what's old. Trivial example here : https://github.com/GafferHQ/gaffer/pull/4807.

Reply:
Switched to angle in 437b0a5